config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
```

Comment:
`:password` was dropped. Intentional?

Reply:
yes, but we don't collect passwords, right?